tests/functional/branching/test_branching.py

```diff
@@ -466,28 +467,66 @@ def _do_nothing(self):
     # If autoresolution was not succesfull, this to fail with a sys.exit without registering the
     # experiment
     orion.core.cli.main(
-        ("init_only -n {name} --branch {branch} --auto-resolution ./black_box_with_y.py "
+        ("init_only -n {name} --branch {branch} ./black_box_with_y.py "
```

Comment:
The patch of BranchingPrompt.cmdloop should not be required anymore. This is because auto-resolution used to prompt user for confirmation even if all conflicts were auto-resolved. We do not prompt anymore.